scripts/utils.js

```diff
@@ -7,6 +7,32 @@ import path from 'node:path';
 import { promises as fs } from 'node:fs';
 import { fileURLToPath } from 'node:url';
 
+const TITLE_TO_SLUG_REPLACEMENTS = {
+  '+': 'plus',
+  '.': 'dot',
+  '&': 'and',
+  đ: 'd',
+  ħ: 'h',
+  ı: 'i',
+  ĸ: 'k',
+  ŀ: 'l',
+  ł: 'l',
+  ß: 'ss',
+  ŧ: 't',
+};
+
+const TITLE_TO_SLUG_CHARS_REGEX = RegExp(
+  // Special characters such as '+' and '.' must be escaped
+  Object.keys(TITLE_TO_SLUG_REPLACEMENTS)
```

Comment:
You don't have to escape them if you use Character Class `[]` instead.

Reply:
But these will not work in the replacement, right? Needs to be pure chars in the replacements object and escaped in the regex string.

Reply:
It can be:

```js
RegExp(`[${Object.keys(TITLE_TO_SLUG_REPLACEMENTS).join('')}]`, 'g')
```

Reply:
Awesome, done some performance tests and it seems to be fast also! 👍🏼